### PR TITLE
Bug 1880379: Clicking on a project link in gitops application details page should navigate to topology view with that project selected

### PR DESCRIPTION
**Pull request by divyanshiGupta**
Jira: https://issues.redhat.com/browse/ODC-4870

Peek 2020-09-23 21-50

/kind bug

### CODE REVIEW

**Review by sahil143 — COMMENTED** [review]
@divyanshiGupta Please add screenshots or gif for the change. It would be better to follow the PR template.

File: frontend/packages/dev-console/src/components/gitops/details/GitOpsDetails.tsx [hunk 1]
```diff
@@ -56,7 +57,21 @@ const GitOpsDetails: React.FC<GitOpsDetailsProps> = ({ envs }) => {
                         )}
                       </StackItem>
                       <StackItem className="co-truncate co-nowrap">
-                        <ResourceLink kind="Project" name={env.environment} />
+                        <span className="co-resource-item">
+                          <ResourceIcon kind="Project" />
+                          {env.cluster ? (
+                            <Link
+                              className="co-resource-item__resource-name"
+                              to={`${env.cluster}/topology/ns/${env.environment}`}
```
Comment (rohitkrai03) [inline frontend/packages/dev-console/src/components/gitops/details/GitOpsDetails.tsx:65]: I think there should be a check to see if the cluster URL is of the current cluster. If yes, then use relative links if not then absolute links. You can check with `route` of `console` deployment in `openshift-console` namespace.

**Review by rohitkrai03 — CHANGES_REQUESTED** [review]

File: frontend/packages/dev-console/src/components/gitops/details/GitOpsDetails.tsx (hunk 1 above)
Comment (divyanshiGupta) [reply frontend/packages/dev-console/src/components/gitops/details/GitOpsDetails.tsx]: I think a normal user will not have access to openshift-console ns :thinking:

**Review by rohitkrai03 — APPROVED** [review]
/approve

**Pull request merged**